thinkhazard/static/less/report.less

```diff
@@ -61,3 +77,48 @@ body {
     }
   }
 }
+
+#map-block .caption.location {
```

Comment:
I'd prefer we use the less nesting (http://lesscss.org/features/#features-overview-feature-nested-rules).
```less
#map-block {
  .caption {
    font-family: @font-semibold;
    text-align: center;
    &.location {
      margin: 0px 20px 27px;
    }
    &.level {
      margin: 4px 20px 0px;
      font-size: 24;
    }
  }
}
```